.changeset/moody-games-wash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@pancakeswap/wagmi': minor
+---
+
+Remove binance extension wallet connector
```

Comment:
@pancakeswap/wagmi: major
@pancakeswap/v3-sdk: patch
@pancakeswap/smart-router: patch
Let's also publich the permit2-sdk and universal router sdk